### ISSUE TITLE
Unable to Navigate Playback Effects Using VoiceOver

### ISSUE THREAD

**Issue by mdrockwell** [Automattic/pocket-casts-ios#1893]
A beta Slack user reported [this issue](https://pocketcastsbeta.slack.com/archives/C012WELJ4VB/p1756744946522169).

**Comment by lizswafford** [comment]
9563712-zd-a8c

> When in the podcast player, tapping the playback effects button brings up a popup that cannot be navigated by swiping on the screen. Finding the items with explore by touch is also incredibly difficult.

App Version: 7.83
Device: iPhone17,5
OS: 18.3.2

**Comment by joashrajin** [comment]
https://forums.pocketcasts.com/forums/topic/the-playback-effects-screen-is-not-accessible-with-voiceover/?view=all

9723483-zd-a8c
